compiler/codegen/symbol.cpp

```diff
@@ -56,6 +54,10 @@
 #include <inttypes.h>
 #include <stdint.h>
 
+#ifdef HAVE_LLVM
+#include "llvm/IR/Attributes.h"
+#endif
```

Comment:
Well, apparently it does, I can recall that it didn't compile without this header but apparently my crazy mind made it up.